databricks/koalas/series.py

```diff
@@ -4258,6 +4258,60 @@ def xs(self, key, level=None):
 
         return _col(DataFrame(internal))
 
+    def pct_change(self, periods=1):
+        """
+        Percentage change between the current and a prior element.
+
+        .. note:: the current implementation of this API uses Spark's Window without
+            specifying partition specification. This leads to move all data into
+            single partition in single machine and could cause serious
+            performance degradation. Avoid this method against very large dataset.
+
+        Parameters
+        ----------
+        periods : int, default 1
+            Periods to shift for forming percent change.
+
+        Returns
+        -------
+        Series
+
+        Examples
+        --------
+
+        >>> kser = ks.Series([90, 91, 85])
+        >>> kser
+        0    90
+        1    91
+        2    85
+        Name: 0, dtype: int64
+
+        >>> kser.pct_change()
+        0         NaN
+        1    0.011111
+        2   -0.065934
+        Name: 0, dtype: float64
+
+        >>> kser.pct_change(periods=2)
+        0         NaN
+        1         NaN
+        2   -0.055556
+        Name: 0, dtype: float64
+        """
+        sdf = self._internal._sdf
+        window = Window.orderBy(self._internal.index_columns).rowsBetween(-periods, -periods)
+
+        for column_name in self._internal.data_columns:
+            prev_row = F.lag(F.col(column_name), periods).over(window)
```

Comment:
`scol_for(sdf, column_name)` instead of `F.col(column_name)`?

Reply:
thanks for the comment :)
at this point, i'm really apologize that i have a question again 😅 

(maybe this question is more related with spark)

for now, when we select from spark dataframe, there are 3 way i recognized.
(let's assume that column name is 'koalas')

1. `sdf.select('koalas')`
2. `sdf.select(F.col('koalas'))`
3. `sdf.select(scol_for(sdf, 'koalas'))`

i know how they are different, but I wonder how it differs in terms of performance. (Perhaps this is why we prefer the 3th way?)

Reply:
The performances are the same.
`scol_for` is introduced for supporting column names including `.` (#217, #490).

Reply:
@ueshin ah, now i have understood after checking that issues. thanks!